jdk/src/share/classes/java/lang/ref/Reference.java

```diff
@@ -245,6 +245,20 @@ public boolean enqueue() {
     }
 
 
+    /**
+     * Throws {@link CloneNotSupportedException}. A {@code Reference} cannot be
+     * meaningfully cloned. Construct a new {@code Reference} instead.
+     *
+     * @returns never returns normally
+     * @throws  CloneNotSupportedException always
+     *
+     * @since 8
```

Comment:
This method is only present in JDK 8u42 (RI) and 8u352 and later releases.  It's not in existing 8u releases >= 8u45 and < 8u352.   Neither 8 or 8u42 would cause confusion.  A suggestion is to use `@apiNote` to indicate which MR this method is defined:

```
@apiNote This method is defined in Java SE 8 Maintenance Release 4.
```

Reply:
The proposed `@apinote` looks good.